### PR TITLE
feat(route): 首席经济学家论坛

### PR DESCRIPTION
**Pull request by kdanfly**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```
/chinacef/portal/hot
/chinacef/17
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/chinacef/experts.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+module.exports = async (ctx) => {
+    const response = await got.get(`http://www.chinacef.cn/index.php/experts/zjmain/experts_id/${ctx.params.experts_id}`);
```
Comment (TonyRL) [inline lib/v2/chinacef/experts.js:7]:
```suggestion
    const response = await got(`http://www.chinacef.cn/index.php/experts/zjmain/experts_id/${ctx.params.experts_id}`);
```
got() use `GET` by default

File: lib/v2/chinacef/maintainer.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
 module.exports = {
-    '/chinacef': ['FledgeXu'],
+    '/': ['FledgeXu'],
+    '/:experts_id?': ['kdanfly'],
```
Comment (TonyRL) [inline lib/v2/chinacef/maintainer.js:3]:
```suggestion
    '/:experts_id': ['kdanfly'],
```

File: lib/v2/chinacef/router.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
 module.exports = function (router) {
+    router.get('/:experts_id?', require('./experts'));
```
Comment (TonyRL) [inline lib/v2/chinacef/router.js:2]:
```suggestion
    router.get('/:experts_id', require('./experts'));
```
`?` in `/:experts_id?` means either 0 or 1 which will conflict with `/`

File: lib/v2/chinacef/experts.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+                        title: $('.contenttitle').first().html().trim(),
+                        article: description,
+                    };
+                });
+
+                return {
+                    title: detail.title,
+                    description: detail.article,
+                    link: itemLink,
+                    pubDate: item.find('span').first().text(),
```
Comment (TonyRL) [inline lib/v2/chinacef/experts.js:40]:
```suggestion
                    pubDate: parseDate(item.find('span').first().text()),
```
const { parseDate } = require('@/utils/parse-date');

File: lib/v2/chinacef/hot.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+                const detail = await ctx.cache.tryGet(itemLink, async () => {
+                    const result = await got.get(itemLink);
+                    const $ = cheerio.load(result.data);
+                    const description = art(path.join(__dirname, 'templates/work_description.art'), {
+                        desc: $('.newsmaintext').first().html().trim(),
+                    });
+                    return {
+                        title: $('.contenttitle').first().html().trim(),
+                        author: $('.zzinfo').find('h2').first().text().trim(),
+                        pubDate: $('.divwidth').text().trim().split(' ')[3],
```
Comment (TonyRL) [inline lib/v2/chinacef/hot.js:31]:
```suggestion
                        pubDate: parseDate($('.divwidth').text().trim().split(' ')[3]),
```

File: lib/v2/chinacef/hot.js [hunk 6]
```diff
@@ -0,0 +1,52 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { art } = require('@/utils/render');
+const path = require('path');
+
```
Comment (kdanfly) [inline lib/v2/chinacef/hot.js:5]: const { parseDate } = require('@/utils/parse-date');

**Comment by kdanfly** [comment]
@TonyRL I'm already fixed the issuses. Thanks for your time.

**Pull request merged**